src/components/Header.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+
+const Header = () => {
+	return (
+		<div className='header'>
+			<h1>BSB<span>player</span></h1>
+		</div>
+	)
+};
```

Comment:
since you are only returning html with no logic, you can return like this
```suggestion
const Header = () =>
	return (
		<div className='header'>
			<h1>BSB<span>player</span></h1>
		</div>
	)
```